### PR TITLE
Fix type in InjectionException

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

File: src/InjectionException.php [hunk 1]
```diff
@@ -8,7 +8,7 @@ final class InjectionException extends InjectorException
      * Add a human readable version of the invalid callable to the standard 'invalid invokable' message.
      *
      * @param array           $inProgressMakes
-     * @param                 $callableOrMethodStr
+     * @param callable        $callableOrMethodStr
```
Comment (kelunik) [inline src/InjectionException.php]: Callable isn't really correct here, because it's really not callable. Maybe mixed?

Comment (szepeviktor) [reply src/InjectionException.php]: What could be in `$callableOrMethodStr`?
Docs say https://www.php.net/manual/en/language.types.callable.php

Comment (szepeviktor) [reply src/InjectionException.php:11]: All these are callable: https://phpstan.org/writing-php-code/phpdoc-types#callables

Comment (szepeviktor) [reply src/InjectionException.php]: I see! As per method name the received value is an invalid callable, which **may** be a callable but not sure!

Comment (szepeviktor) [reply src/InjectionException.php]: There is no 'else' only 'elseif'.

Comment (szepeviktor) [reply src/InjectionException.php]: Done.

**Pull request merged**